components/mdns/include/mdns.h

```diff
@@ -21,6 +21,13 @@ extern "C" {
 #define MDNS_TYPE_NSEC              0x002F
 #define MDNS_TYPE_ANY               0x00FF
 
+#if defined(CONFIG_LWIP_IPV6) && defined(CONFIG_MDNS_RESPOND_REVERSE_QUERIES)
```

Comment:
Why make this public?

Reply:
since `mdns_hostname_get()` lets user to allocate a buffer this maximum size